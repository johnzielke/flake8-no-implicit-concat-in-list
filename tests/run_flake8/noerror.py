a = "aaa"
b = "bbb" + "ccc"
d = """ddd
eee
fff""" + "ggg"
l = ["hhh",
     "iii" +
     "jjj"]
noqa = "k" "l"
noqa_2 = ("k"
          "l")
noqa_3 = ("m"  # noqa: ICL211
          "n",)
print("abc" + "def", "ghi")
