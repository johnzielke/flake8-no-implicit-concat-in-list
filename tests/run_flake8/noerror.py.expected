./noerror.py:12:14: ICL221 Implicitly concatenated str literals in tuple literal over multiple lines.
